integration_tests/__tests__/coverage_remapping.test.js

```diff
@@ -29,7 +29,7 @@ it('maps code coverage against original source', () => {
 
   // reduce absolute paths embedded in the coverage map to just filenames
   Object.keys(coverageMap).forEach(filename => {
-    coverageMap[filename].path = path.basename(coverageMap[filename].path);
+    coverageMap[filename].data.path = path.basename(coverageMap[filename].data.path);
```

Comment:
That's the shady change. It's like we were using a completely different version of the "coverage module", whichever it is. See also how the snap file is pretty different.